io.github.iSlammedMyKindle.godotGem.yml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.iSlammedMyKindle.godotGem
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: godotGemClient.x86_64
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+modules:
+  - name: godotGem
+    buildsystem: simple
+    build-commands:
+      - install -D godotGemClient.x86_64 $FLATPAK_DEST/bin/godotGemClient.x86_64
+      - install -D $FLATPAK_ID.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -D icon.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
+      - install -D icon.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -D $FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: archive
+        url: https://github.com/iSlammedMyKindle/godotGem/releases/download/1.0/godotGemClient.zip
+        sha256: 8a399df2e696dca134038c3bb79a890f74d65f65a2872bca072c30562ccd21ad
+      - type: file
+        path: icon.png
+      - type: file
+        path: icon.svg
+      - type: file
+        path: io.github.iSlammedMyKindle.godotGem.desktop
+      - type: file
+        path: io.github.iSlammedMyKindle.godotGem.metainfo.xml
```

Comment:
Please consider making these files part of the zip file above in the future releases.